jobserver/models.py

```diff
@@ -371,6 +352,33 @@ def save(self, *args, **kwargs):
         return super().save(*args, **kwargs)
 
 
+class OrgMembership(models.Model):
+    created_by = models.ForeignKey(
+        "User",
+        on_delete=models.SET_NULL,
+        related_name="org_invitations",
```

Comment:
I think this is a bit ambiguous -- does `george.org_invitations.all()` return all the `OrgMembership`s that George has invited people to, or all the `OrgMembership`s that George has been invited to?  Given that we're modelling more than just an invitation here, what about something like `related_name="created_org_memberships"`?